examples/pipelines/starcoder/components/comments_filtering/fondant_component.yaml

```
@@ -0,0 +1,18 @@
+name: Comments filtering
+description: Component that filters scripts based on the code to comment ratio
+image: ghcr.io/ml6team/comment_filtering:latest
```

```suggestion
image: ghcr.io/ml6team/filter_comments:latest
```
I see the other component is called filter_metadata so let's use consistent names